tee-worker/core/tls-websocket-server/src/connection.rs

```diff
@@ -189,14 +218,16 @@ where
 				);
 			},
 			Message::Binary(_) => {
-				warn!("received binary message, don't have a handler for this format");
+				unreachable!()
```

Comment:
I have a question here, Do you think it makes sense to panic here?

Reply:
Ah good question, I forgot to change it back since I wanted to hard panic here when debugging.